core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/model/GlobalDefinitionCamelModelElement.java

```diff
@@ -26,4 +26,24 @@ public GlobalDefinitionCamelModelElement(AbstractCamelModelElement parent, Node
 		super(parent, underlyingNode);
 	}
 
+	/* (non-Javadoc)
+	 * @see org.fusesource.ide.camel.model.service.core.model.AbstractCamelModelElement#parseNode()
+	 */
+	@Override
+	protected void parseNode() {
+	}

```

Comment:
isn't parseNode enough to override?
Maybe we should add a comment in the empty method to explain why it is empty. something like: "no parsing, reuse XML Node directly"